app/js/trouble.js

```diff
@@ -1,22 +1,24 @@
-'use strict';
 
-(function (gbl, $) {
+
+function foo (gbl, $) {

```

Comment:
Leave the '(' in place here as this allows it to be an immediately invoked function expression.